src/mad_spark_alt/core/unified_evaluator.py

```
@@ -0,0 +1,274 @@
+            return HypothesisEvaluation(
+                content=hypothesis,
+                scores=scores,
+                overall_score=overall_score,
+                explanations=explanations,
+                metadata={
+                    "llm_cost": response.cost,
+                    "model": response.model
+                }
+            )
+            
+        except Exception as e:
```

Ruff (0.12.2)

Do not catch blind exception: `Exception`

(BLE001)